lib/coverage-instrumenter.js

```diff
@@ -16,12 +56,18 @@ CoverageInstrumenter.prototype.extensions = ['js'];
 CoverageInstrumenter.prototype.targetExtension = 'js';
 
 CoverageInstrumenter.prototype.processString = function(content, relativePath) {
-  var instrumenter = new Istanbul.Instrumenter({
+  var instrumenter = new Instrumenter({
+    babel: {
+      blacklist: ['es6.modules'],
+      nonStandard: false,
+      optional: ['es7.decorators']
```

Comment:
This seems odd to hard code. Ultimately, we need to pass through whatever the app being compiled is using for its plugins...